### PR TITLE
Add a highlights section to changelog.

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Purpose of change
Add a highlights section to changelog for 0.Funnamed

Lemme know if there are any super impactful features you think belong in here, there's room for a few more.

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+Weariness tracking added to represent longer term physical exhaustion.
+Elevated Bridges/Navigable Rivers added, allowing better navigabilty while using boats.
+Large scale audit of weapon and armor values for better representativeness and consistency.
+Improved armor handling by seperating ballistic damage into it's own damage type.
+Pervasive performance enhancements throughout the game.
+Tileset vehicle support for more cohesive vehicle rendering.
+Aftershock changing direction to a total conversion with a new far sci-fi setting set on a frozen world.
+Dinomod added 238 dinosaurs, pterosaurs, mosasaurs, dino-related NPCs with missions and dino locations.
+Added many dino features, including zombie, fungal, evolved, bionic, baby, and mutant dino variants.
+Added many dino interactions, including farming, riding, butchering, cooking, and special attacks.
```
Comment (KorGgenT) [inline data/changelog.txt:17]:
```suggestion
Added many dino interactions, including farming, riding, butchering, cooking, and special attacks.
Magiclysm added a huge content update with many new traits called Attunements that switch up gameplay at the endgame.
```

File: data/changelog.txt [hunk 2]
```diff
@@ -1,5 +1,22 @@
 # 0.F (2021-06-19)
 
+## Highlights
+Nested Containers rationalize inventory management and enable dropping and retrieving go-bags during fights.
+Achievements track your deeds and misdeeds across games.
+Proficiencies better represent deeper knowledge required for various endeavors, mostly crafting.
+Bleeding added to both the player and monsters as the first step toward a more comprehensive wound and wound treatment system.
```
Comment (Night-Pryanik) [inline data/changelog.txt:7]:
```suggestion
Bleeding added to both the player and monsters as the first step toward a more comprehensive wound and wound treatment systems.
```

File: data/changelog.txt [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+Weariness tracking added to represent longer term physical exhaustion.
+Elevated Bridges/Navigable Rivers added, allowing better navigabilty while using boats.
```
Comment (Night-Pryanik) [inline data/changelog.txt:9]:
```suggestion
Elevated bridges/navigable rivers added, allowing better navigabilty while using boats.
```

File: data/changelog.txt [hunk 4]
```diff
@@ -1,5 +1,22 @@
...
+Large scale audit of weapon and armor values for better representativeness and consistency.
+Improved armor handling by seperating ballistic damage into it's own damage type.
```
Comment (Night-Pryanik) [inline data/changelog.txt:11]:
```suggestion
Improved armor handling by separating ballistic damage into its own damage type.
```

File: data/changelog.txt [hunk 5]
```diff
@@ -1,5 +1,22 @@
...
+Pervasive performance enhancements throughout the game.
+Tileset vehicle support for more cohesive vehicle rendering.
+Aftershock changing direction to a total conversion with a new far sci-fi setting set on a frozen world.
+Dinomod added 238 dinosaurs, pterosaurs, mosasaurs, dino-related NPCs with missions and dino locations.
```
Comment (actual-nh) [inline data/changelog.txt:15]:
```suggestion
Aftershock changes direction to a total conversion mod with a new far sci-fi setting set on a frozen world.
Dinomod added 238 dinosaurs, pterosaurs, mosasaurs, and dino-related NPCs with missions and dino locations.
```
Not sure about "far sci-fi".

File: data/changelog.txt (hunk 1 above)
Comment (actual-nh) [reply data/changelog.txt:17]: "Magiclysm added a huge content update including many new traits called Attunements that switch up gameplay at the endgame."

File: data/changelog.txt (hunk 2 above)
Comment (actual-nh) [reply data/changelog.txt:7]: Would it be different systems or one system?

**Review by actual-nh — COMMENTED** [review]
Looking good!

File: data/changelog.txt (hunk 2 above)
Comment (actual-nh) [reply data/changelog.txt:7]: Would it be different systems or one system?

File: data/changelog.txt (hunk 1 above)
Comment (actual-nh) [reply data/changelog.txt:17]: "Magiclysm added a huge content update including many new traits called Attunements that switch up gameplay at the endgame."

File: data/changelog.txt (hunk 5 above)
Comment (KorGgenT) [reply data/changelog.txt:15]: possibly "far future"?

File: data/changelog.txt (hunk 1 above)
Comment (anothersimulacrum) [reply data/changelog.txt:17]:
```suggestion
Added many dino interactions, including farming, riding, butchering, cooking, and special attacks.
Magiclysm added a huge content update including many new traits called Attunements that switch up gameplay at the endgame.
```

File: data/changelog.txt (hunk 2 above)
Comment (anothersimulacrum) [reply data/changelog.txt:7]: Need to drop the `a` if it's systems.
```suggestion
Bleeding added to both the player and monsters as the first step toward more comprehensive wound and wound treatment systems.
```

File: data/changelog.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (anothersimulacrum) [reply data/changelog.txt:9]:
```suggestion
Elevated bridges over navigable rivers added, allowing better navigability while using boats.
```

File: data/changelog.txt (hunk 5 above)
Comment (anothersimulacrum) [reply data/changelog.txt:15]:
```suggestion
Aftershock changes direction to a total conversion mod with a new far-future sci-fi setting set on a frozen world.
Dinomod added 238 dinosaurs, pterosaurs, mosasaurs, and dino-related NPCs with missions and dino locations.
```

File: data/changelog.txt (hunk 3 above)
Comment (anothersimulacrum) [reply data/changelog.txt:9]: Combination of this and actual-nh's suggestions: https://github.com/CleverRaven/Cataclysm-DDA/pull/49501#discussion_r661132911

File: data/changelog.txt (hunk 1 above)
Comment (PatrikLundell) [reply data/changelog.txt:17]: Shouldn't each bullet that applies to a specific mod mention the mod? In the list at the top of this section Dinomod is mentioned for the first point, but I'd expect the two following ones to apply to the same mod only.

Comment (anothersimulacrum) [reply data/changelog.txt:17]:
```suggestion
Dinomod added many dino interactions, including farming, riding, butchering, cooking, and special attacks.
Magiclysm added a huge content update including many new traits called Attunements that switch up gameplay at the endgame.
```

File: data/changelog.txt (hunk 5 above)
Comment (actual-nh) [reply data/changelog.txt:15]:
```suggestion
Aftershock changes direction to a total conversion mod with a new far-future setting on a frozen world.
Dinomod added 238 dinosaurs, pterosaurs, mosasaurs, and dino-related NPCs with missions and dino locations.
```

File: data/changelog.txt (hunk 2 above)
Comment (kevingranade) [reply data/changelog.txt:7]: It's singular, "wound and wound treatment" "system".

**Pull request merged**